src/tools/rust-analyzer/crates/stdx/src/anymap.rs

```diff
@@ -79,9 +77,6 @@ pub type RawMap<A> = hash_map::HashMap<TypeId, Box<A>, BuildHasherDefault<TypeId
 ///   also spelled [`AnyMap`] for convenience.
 /// - <code>[Map]&lt;dyn [core::any::Any] + Send&gt;</code>
 /// - <code>[Map]&lt;dyn [core::any::Any] + Send + Sync&gt;</code>
-/// - <code>[Map]&lt;dyn [CloneAny]&gt;</code>
```

Comment:
Can you pls pull this r-a change into a separate PR?

Reply:
Opened https://github.com/rust-lang/rust-analyzer/pull/17562

Reply:
Thanks. I left a comment asking if that PR would unnecessarily block this PR -- worst case we can do this in-tree with r-a approval.